### PR TITLE
refactor release job to run over GH actions instead of cloudbuild

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- refactor release job to run over GH actions instead of cloudbuild

reduce a bit the complexity and make it simpler to run

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@84cbf8094393cdc5fe1fe1671ff2647332956b1a # v3
+        with:
+          go-version: 1.18
+          check-latest: true
+
+      - uses: sigstore/cosign-installer@09a077b27eb1310dcfb21981bee195b30ce09de0 # v2
+      - uses: anchore/sbom-action/download-syft@bb716408e75840bbb01e839347cd213767269d4a # v0.11.0
```
Comment (hectorj2f) [inline .github/workflows/release.yaml:25]: thanks, we needed this one.

**Review by hectorj2f — APPROVED** [review]
lgtm

**Comment by cpanato** [comment]
ping @vaikas

**Review by vaikas — APPROVED** [review]
I'm so sorry for the tardiness, for some reason in my head I thought it was still a draft and you were working on it.

**Pull request merged**